modules/account-lib/src/coin/cspr/transactionBuilderFactory.ts

```diff
@@ -31,7 +33,14 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
     if (tx.casperTx.session.isTransfer()) {
       return this.getTransferBuilder(tx);
     } else if (tx.casperTx.session.isModuleBytes()) {
-      // TODO(stlx-1458), identify wallet initialization deploy
+      const moduleBytes = tx.casperTx.session.getArgByName('moduleBytes');
+      if (moduleBytes) {
+        const contract = Uint8Array.from(Buffer.from(walletInitContractHexCode, 'hex'));
+        console.log(moduleBytes.asBytesArray() === contract);
+      } else {
```

Comment:
Are we confident that the `raw` argument always has a `session` property with the methods`isTransfer()`, `isModuleBytes()` and `getArgByName()`?
Otherwise we should check and throw the appropriate exception.

Reply:
Yes, we're confident on that.
To create a Deploy object an ExecutableDeployItem must be set as "session". Every ExecutableDeployItem has a method isTransfer() and isModuleBytes() and also getArgByName(...)